packages/@aws-cdk/aws-applicationsignals-alpha/lib/slo/constants.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Types of metrics that can be used for SLIs
+ */
+export enum MetricType {
+    /**
+     * Latency-based metric type
+     * Used for measuring response time or duration
+     */
+    LATENCY = 'LATENCY',
+
+    /**
+     * Availability-based metric type
+     * Used for measuring uptime or success rate
+     */
+    AVAILABILITY = 'AVAILABILITY'
+}
+
+/**
+ * Default values for goal configuration
+ */
+export const DEFAULT_GOAL_CONFIG = {
+    ATTAINMENT_GOAL: 99.9,
+    WARNING_THRESHOLD: 30,
+} as const;
```

Comment:
Either move these constants as static properties in some class and it is better to be the classes where these values be used, or make them private. Do we need to add these values in CDK Docs, and let the customers use them?